notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>proj does select the primary keys but the main reason for using it is to select a certain entry from Selection table. if we didn't have it and just ran Recording.populate(), it would operate on every entry of Selection table. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>I might be a bit confused given that the ssr_key should be what selects the entry from Selection and proj is just passing the arguments from Selection to Recording.populate(), right? "so it knows exactly what to extract" is less programmatic and more conversational. Happy to change if you think it conveys the wrong meaning. </p>

Reply:
<p>yes you're right - sorry ignore that comment. I also don't know why it is showing up here rather than in the one below...</p>